src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,35 @@
+import {Link} from "react-router-dom";
+import {Carousel, Button} from "antd";
+import {categories} from '../../types';
+import main1 from '../../images/main1.jpg';
+import main2 from '../../images/main2.jpg';
+import css from './menu.module.css';
+
+export const Menu = () => {
+    return (
+        <div className={css.container}>
+            <ul className={css.list}>
+                {categories.map((category) => (
+                    <Link to={`/${category.type}`} key={category.id} className={css.categories}>
+                        <li style={{marginTop: 15}}>{category.label}</li>
+                    </Link>
+                ))}
+            </ul>
+            <div className={css.carousel_block}>
```

Comment:
именуем через кемел кейс
```suggestion
            <div className={css.carouselBlock}>
```